cmd/erasure-sets.go

```diff
@@ -804,16 +819,21 @@ func (s *erasureSets) DeleteObjects(ctx context.Context, bucket string, objects
 		objSetMap[index] = append(objSetMap[index], delObj{setIndex: index, origIndex: i, object: object})
 	}
 
+	var backendServers []string
+
 	// Invoke bulk delete on objects per set and save
 	// the result of the delete operation
 	for _, objsGroup := range objSetMap {
-		dobjects, errs := s.getHashedSet(objsGroup[0].object.ObjectName).DeleteObjects(ctx, bucket, toNames(objsGroup), opts)
+		set := s.getHashedSet(objsGroup[0].object.ObjectName)
+		backendServers = append(backendServers, set.getEndpoints()...)
+		dobjects, errs := set.DeleteObjects(ctx, bucket, toNames(objsGroup), opts)
 		for i, obj := range objsGroup {
 			delErrs[obj.origIndex] = errs[i]
 			delObjects[obj.origIndex] = dobjects[i]
 		}
 	}
 
+	logBackendServers(ctx, backendServers)
```

Comment:
We should also provide a map of `objectName->backendServers` here otherwise its confusing that a large list of servers is involved if the objects are located across many sets - lets say in a 3000 drive setup.

Reply:
> We should also provide a map of `objectName->backendServers` here otherwise its confusing that a large list of servers is involved if the objects are located across many sets - lets say in a 3000 drive setup.

Well, if we are doing this, then we cannot use logger.SetTags and we should add another logger API. Frankly I feel this is an overkill, I am not sure why would someone needs the mapping between object and backend servers

Reply:
That is the main information customer wants @vadmeste - it needs to be per object - otherwise this list is going to be 1024 disks in some setups and it won't be meaningful to parse.

You need a double inference such as each audit log for DeleteMultipleObjects should carry how many objects it deleted separately because currently, we don't audit that information in DeleteMultipleObjects. 

```
type ReqInfo
```

Can have `Objects` key that is only set for this special API and its empty for all other APIs